util/trim.go

```diff
@@ -5,8 +5,14 @@ package util
 
 import "strings"
 
-// TrimQuote returns a slice of the string s, with all leading
+// TrimQuotes returns a slice of the string s, with all leading
 // and trailing double or single quotes removed, as defined by Unicode.
 func TrimQuotes(s string) string {
 	return strings.Trim(s, "'\"")
 }
+
+// TrimArtifactPrefix returns a slice of the string s, with leading
+// artifact prefix removed
+func TrimArtifactPrefix(s string) string {
+	return strings.TrimPrefix(s, "oci://")
```

Comment:
>oci:// [](start = 31, length = 6)

Is the schema like `OCI://`, "HTTPS://" valid? If yes, can you handle it in a case insensitive way?